radiation/FatesRadiationDriveMod.F90

```diff
@@ -282,17 +279,15 @@ subroutine FatesSunShadeFracs(nsites, sites,bc_in,bc_out)
 
     do s = 1,nsites
 
-       ifp = 0
        cpatch => sites(s)%oldest_patch
-
        do while (associated(cpatch))
 
-          if_notbareground:if(cpatch%nocomp_pft_label.ne.nocomp_bareground)then !only for veg patches
-             ! do not do albedo calculations for bare ground patch in SP mode
-             ! and (more impotantly) do not iterate ifp or it will mess up the indexing wherein
-             ! ifp=1 is the first vegetated patch.
-             ifp=ifp+1
+          ifp = cpatch%patchno
+          
+          nocomp_bareground:if(cpatch%nocomp_pft_label.ne.nocomp_bareground)then !only for veg patches
```

Comment:
The `end if` side of this block still shows `if_notbareground`.

Reply:
actually, this will not compile on gnu compilers, I will def have to fix this

Reply:
yeah, gnu compilers gave me trouble if I used a label that is also a variable. I think its a reasonable complaint. Reverted this loop label back to if_bareground.